tests/wast.rs

```diff
@@ -78,6 +78,10 @@ fn ignore(test: &Path, strategy: Strategy) -> bool {
         if part == "extended-const" {
             return true;
         }
+        // Wasmtime doesn't implement the table64 extension yet.
+        if part == "memory64" {
+            return true;
+        }
```

Comment:
It seems a bit surprising that we have to ignore *all* memory64 tests now and not just the table related ones. Is the latter possible?